cypress/integration/home.spec.ts

```diff
@@ -24,8 +28,13 @@ describe('Home screen', () => {
       cy.visit('/');
 
       // Click on the first post
-      cy.findByTestId(homeDataTestIds.postLink).first().click();
+      cy.findAllByTestId(homeDataTestIds.postLink).first().click();
 
+      cy.get('h1', { timeout: 15000 }).should(
```

Comment:
@hoangmirs is this `{ timeout: 15000 }` because of `remark.js`? 🫣

Reply:
Yes, It seems RemarkJS renders quite slowly